Potentiometer_THT.pretty/Potentiometer_Alpha_RD901F-40-00D_Single_Vertical.kicad_mod

```diff
@@ -0,0 +1,41 @@
+(module "Potentiometer_Alpha_RD901F-40-00D_Single_Vertical" (layer F.Cu) (tedit 5C69D274)
```

Comment:
@robkam, when you've modified and then fixed the names, some quotation marks `" "` have been added. The `module` line doesn't use quotations (you can check around in the library). Can you fix that? Thanks!

Reply:
This has been fixed now. Thanks.